cmd/skywire-cli/commands/visor/start.go

```diff
@@ -0,0 +1,45 @@
+package visor
+
+import (
+	"os/user"
+
+	"github.com/bitfield/script"
+	"github.com/spf13/cobra"
+)
+
+var sourcerun bool
+
+func init() {
+	usrLvl, err := user.Current()
+	if err != nil {
+		panic(err)
+	}
+	var root bool
+	if usrLvl.Username == "root" {
+		root = true
+	}
+	//enforce root permissions for VPN operability
```

Comment:
Do we need to enforce that? Should that not be done by the package automatically?

Reply:
The package does not currently use this command directly in any script.

Binaries are installed to /usr/bin not sbin

as far as i knew, it needed to be enforced but i have no mechanism for doing so in the package itself.

this may need to be changed if we can get rootless vpn working

Reply:
changed to use the appropriate config for the level you invoke the command with

* It does not check that the config exists already.

if you use root, it uses the config generated by `skywire-cli config gen -p`

if you run as the user, it uses the config generated by `skywire-cli config gen -u`